pastis/pastis_analysis.py

```diff
@@ -132,6 +132,18 @@ def full_modes_from_themselves(instrument, pmodes, datadir, sim_instance, saving
             jwst_wavenumber = 2 * np.pi / (CONFIG_PASTIS.getfloat('JWST', 'lambda') / 1e9)   # /1e9 converts to meters
```

Comment:
It would be good if you could revert all changes made to this file. I think we have talked about how you can revert them but still keep them for later use. We can talk about this offline if you want.